x-pack/.i18nrc.json

```diff
@@ -39,7 +39,8 @@
     "xpack.transform": "legacy/plugins/transform",
     "xpack.upgradeAssistant": "legacy/plugins/upgrade_assistant",
     "xpack.uptime": "legacy/plugins/uptime",
-    "xpack.watcher": "legacy/plugins/watcher"
+    "xpack.watcher": "legacy/plugins/watcher",
+    "xpack.tasks": "legacy/plugins/es_task_management"
```

Comment:
We might want to check in with @elastic/kibana-stack-services and make sure we're not hijacking a namespace they wanted for themselves.